<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchTextDialog</class>
 <widget class="QDialog" name="SearchTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find or replace</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="findEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="findLabel">
     <property name="text">
      <string>Find:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="3" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="caseSensitiveCheck">
        <property name="text">
         <string>Case sensitive</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="backwardsCheck">
        <property name="text">
         <string>Search backwards</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="regExpCheck">
        <property name="text">
         <string>Regular expression matching</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="replaceButton">
     <property name="text">
      <string>Replace &amp;&amp;
find next</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string>Replace with:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="replaceEdit"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="replaceAllButton">
     <property name="text">
      <string>Replace all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="findButton">
     <property name="text">
      <string>Find</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>replaceEdit</tabstop>
  <tabstop>caseSensitiveCheck</tabstop>
  <tabstop>regExpCheck</tabstop>
  <tabstop>backwardsCheck</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchTextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchTextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
